diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index 7900f5f23e..f44e121af0 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -102,2 +102,6 @@ source_set("configs") {
 
+  if (is_linux || is_win) {
+    deps += [ "//prism/ui/webui:prism_ai_ui" ]
+  }
+
   if (is_win || is_chromeos) {
diff --git a/chrome/browser/ui/webui/chrome_web_ui_configs.cc b/chrome/browser/ui/webui/chrome_web_ui_configs.cc
index 093ceae662..f078aced0a 100644
--- a/chrome/browser/ui/webui/chrome_web_ui_configs.cc
+++ b/chrome/browser/ui/webui/chrome_web_ui_configs.cc
@@ -69,2 +69,5 @@
 #include "printing/buildflags/buildflags.h"
+#if BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_WIN)
+#include "prism/ui/webui/prism_ai_ui.h"
+#endif
 #include "ui/webui/buildflags.h"
@@ -271,2 +274,5 @@ void RegisterChromeWebUIConfigs() {
   auto& map = content::WebUIConfigMap::GetInstance();
+#if BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_WIN)
+  map.AddWebUIConfig(std::make_unique<prism::ui::PrismAiUIConfig>());
+#endif
   map.AddWebUIConfig(std::make_unique<AccessibilityUIConfig>());
